__tests__/index.js

```diff
@@ -30,14 +30,15 @@ async function execCommand(
   }
 
   return new Promise((resolve, reject) => {
+    const {...cleanedEnv} = {...process.env};
```

Comment:
Isn't `const cleanedEnv = {...process.env};` enough?

Reply:
You're right! Thank you very much for the hint.